powerquery-docs/Connectors/DatabricksCloud.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Databricks connector
+description: Provides basic information and prerequisites for using Power Query's Databricks connectors.
+author: George Chow
+
+ms.topic: conceptual
+ms.date: 6/29/2022
+ms.author: George Chow
+LocalizationGroup: reference
+---
+
+# Databricks Cloud
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | Azure Databricks: General Availability<br/>Databricks: Preview |
+| Products | Power BI (Datasets)<br/>Power BI (Dataflows) |
+| Authentication Types Supported | Azure Active Directory<br/>Organizational Account<br/>Personal Access Token |
+| | |
+
+>[!Note]
+>Some capabilities may be present in one product but not others due to deployment schedules and host-specific capabilities.
+
+
+## Prerequisites
+
+You requires either an Azure Databricks account or a paid Databricks Cloud account to use this connector. Databricks Community Edition is not supported.
+
+## Capabilities supported
+
+* Import
+* DirectQuery (PowerBI only)
+
+## Connect to Azure Databricks or Databricks Cloud from Power Query Desktop
+
+To connect from Power BI Desktop, take the following steps:
+
+1. In the Get Data experience, search for **databricks** to shortlist the two Databrick connectors: "Azure Databricks" and "Databricks". Choose "Azure Databricks" if your Databricks runs in the Azure cloud; for AWS or GCP, choose "Databricks". (If the entry for "Databricks" is marked with a "(Beta)" suffix, contact your Databricks representative for guidance on using the connector.)
+
+    ![Get Data from Databricks Cloud.](./media/databricks-cloud/get-data-dbc.png)
+
+2. You will need to provide the "Server hostname" and "HTTP Path" for your Databricks SQL Warehouse. Refer [here](https://docs.microsoft.com/en-us/azure/databricks/integrations/bi/jdbc-odbc-bi#get-server-hostname-port-http-path-and-jdbc-url) for instructions to look up your "Server hostname" and "HTTP Path". Enter these accordingly. You may optionally supply a default catalog and/or database under **Advanced options**. Select **OK** to continue.
+
+    ![Specify your Databricks SQL Warehouse.](./media/databricks-cloud/azdbc-sql-endpoint.png)
+
+3. You will need provide your credentials to authenticate with your Databricks SQL Warehouse. For Azure Databricks, you can choose between 1) basic user/password; 2) Personal Access Token; and 3) Azure Active Directory (AAD). For AWS or GCP, your choice is between 1) basic user/password; 2) Personal Access Token; and 3) OAuth. Refer [here](https://docs.microsoft.com/en-us/azure/databricks/sql/user/security/personal-access-tokens) for instructions on generating a Personal Access Token (PAT). For organizational account (when using Azure Databricks), you will be prompted to log into your Azure Databricks cluster. For OAuth (when using Databricks on AWS or GCP), you will be prompted to log into your Databricks SQL Warehouse.
```

Comment:
as mentioned above, till we remove the beta tag, the public documentation should promote the use of the Azure Databricks connector for prod even for AWS and GCP. We should just briefly point out the new beta generic connector.

also please note the generic beta connector has support for OAuth only on AWS but not for GCP.

Reply:
k... corrected.